dom/abort/event.any.js

```diff
@@ -139,4 +139,22 @@ test(t => {
   assert_equals(signal.reason, reason, "signal.reason");
 }, "static aborting signal with reason should set signal.reason");
 
+test(t => {
+  const reason = new Error('boom');
+  const signal = AbortSignal.abort(reason);
+  assert_true(signal.aborted);
+  try {
+    signal.throwIfAborted();
+    throw new Error("failed to throw when expected");
+  } catch (err) {
+    assert_equals(err, reason);
+  }
```

Comment:
You want https://web-platform-tests.org/writing-tests/testharness-api.html#assert_throws_exactly.
```suggestion
  assert_throws_exactly(reason, () => signal.throwIfAborted());
```